frontend/src/metabase/admin/datamodel/metadata/components/MetadataTable/MetadataTable.tsx

```diff
@@ -142,7 +134,7 @@ const MetadataTableView = ({
   );
 
   const handleChangeDescription = useCallback(
-    (description: string) => {
+    (description: string | null) => {
```

Comment:
nit: can we use `undefined` instead of `null` for types? I'm curiuos if `null` covers some specific case, when it's not initial "empty" value but at the the result of removed description

Reply:
`null` is what should be sent to the backend